api/src/wfl/service/google/bigquery.clj

```diff
@@ -46,6 +46,13 @@
       util/response-body-json
       :tables))
 
+(defn ^:private normalize-table [{:keys [schema] :as response}]
+  (let [repeated? (mapv #(= "REPEATED" (:mode %)) (:fields schema))]
```

Comment:
No, this is because elements in arrays returned by big query are objects `[{:v x}]`. The original flatten implementation code didn't extract those values. I'm testing if the column is an array (or REPEATED in big query speak)

Reply:
Ah, got it, makes sense!